sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/FlatMapElementsTest.java

```diff
@@ -103,6 +106,19 @@ public void testFlatMapFnOutputTypeDescriptor() throws Exception {
   }
 
   @Test
+  public void testSimpleFunctionClassDisplayData() {
```

Comment:
It seems like there should be tests for the "propagate input coder to output" logic in at least a few diffferent cases T -> T, T -> Iterable<T>, etc.